server/channels/app/email/email.go

```diff
@@ -1275,3 +1275,40 @@ func (es *Service) SendRemoveExpiredLicenseEmail(ctaText, ctaLink, email, locale
 
 	return nil
 }
+
+func (es *Service) SendIPFiltersChangedEmail(email string, initiatingUser *model.User, siteURL, portalURL, locale string, isWorkspaceOwner bool) (bool, error) {
```

Comment:
We're not using the first return value `bool`, do we still need to keep it for consistency or some future use?

Reply:
The file's probably 60/40 on not having the bool return value so I removed it

Reply:
Got it.